skins/laika/src/view_models/Address.ts

```diff
@@ -32,6 +32,7 @@ export interface AddressState {
     receiptOptOut: boolean,
     values: FormValues,
     validity: FormValidity,
+    requiredFields: { [key: number]: string[] },
```

Comment:
probably should be it's own interface 'cause it looks confusing. Especially when you repeat it in `address/index.ts`